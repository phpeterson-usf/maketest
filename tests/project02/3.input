-e "10" -b 2
