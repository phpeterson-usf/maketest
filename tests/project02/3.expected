0111 1111 1111 1111 1111 1111 1111 1111 (base 2)
0b01111111111111111111111111111111 (base 2)
0x7FFFFFFF (base 16)
2147483647 (base 10 unsigned)
2147483647 (base 10 signed)
